baselines/bench/monitor.py

```diff
@@ -56,26 +59,29 @@ def step(self, action):
         return (ob, rew, done, info)
 
     def update(self, ob, rew, done, info):
+        self.total_steps += 1
         self.rewards.append(rew)
         if done:
             self.needs_reset = True
             eprew = sum(self.rewards)
             eplen = len(self.rewards)
-            epinfo = {"r": round(eprew, 6), "l": eplen, "t": round(time.time() - self.tstart, 6)}
+            ep_t = time.time() - self.tstart
+            ep_dt = time.time() - self.episode_start
```

Comment:
dt is the episode duration, correct? Is is really necessary (given that it can be deduced from differentiating t)?

Reply:
For the custom environment, I was working with `ep_dt` wasn't easily deducible from `ep_t` as there were few things happening before the next episode start. However, I understand this is probably not the case for most other environments so I can remove logging this one if needed.